### DISCUSSION TITLE
Add successful response schemes for auth endpoints

### DISCUSSION THREAD

**Discussion by ysomad** [fastapi-users/fastapi-users#577] [Ideas]
Would be great if you can add successful response schemes for:
1. login/
2. forgot-password/
3. reset-password/
4. request-verify-token/

**Comment by frankie567** [comment]
Do you mean schemas so that you have it in the interactive Swagger documentation?

PS: Please favor **Discussions** over issues when not reporting a bug.

**Reply by ysomad** [reply]
Yes!

My bad
